scripts/netContain/scripts/contivNet.sh

```diff
@@ -133,19 +132,21 @@ if [ $netmaster == true ]; then
 	done
 elif [ $netplugin == true ]; then
 	echo "Starting netplugin"
+    if [[ "$cluster_store" =~ ^etcd://.+ ]]; then
+        store_arg="--etcd-endpoints $(echo $cluster_store | sed s/etcd/http/)"
+    elif [[ "$cluster_store" =~ ^consul://.+ ]]; then
+        store_arg="--consul-endpoints $(echo $cluster_store | sed s/consul/http/)"
```

Comment:
Can `--cluster-store` and `--listen-url` be retained for backward compatibility?

Reply:
it will still work for the container script but not for binary, because `--cluster-store` has restriction on the url schema